.github/ISSUE_TEMPLATE.md

```diff
@@ -1,31 +1,41 @@
-- Did you search for similar issues before submitting this one?
+<!--
+Before submitting this issue, please visit our wiki for common ones: https://github.com/brave/browser-laptop/wiki
 
-- Describe the issue you encountered:
+Have you searched for similar issues? There is a lot of feedbacks and bug reports we have received and closed as duplicate.
+By using search engines along with GitHub search function you would be able to find duplicates more efficiently.
 
-- Platform (Win7, 8, 10? macOS? Linux distro?):
+For more, check out our community site: https://community.brave.com/
```

Comment:
I was going to add a link to our discord channel, which on the second thought seemed too much. If it isn't, I'll add that.

Reply:
Not sure if the first line about visiting wiki makes sense at top.  I would have it like this 

```
Have you searched for similar issues? There is a lot of feedback and bug reports that we have received and closed as duplicate.  
Please search for existing issue via GitHub search or using a search engines. 

Please check our community site: https://communit.brave.com and also have a look at our Wiki: https://github.com/brave/browser-laptop/wiki/
```

Reply:
> Not sure if the first line about visiting wiki makes sense at top.

I'd agree. I'll edit this comment block.